testing-with-jest.test.js

```diff
@@ -6,22 +6,84 @@
 // $ yarn jest
 
 
-// --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
-
-
-
-
-
-// --------------------1b) See the test fail. THEN write the code to make the test pass.
-
-
-
-
-
-// --------------------2a) Create a test for a function called triangleArea that takes the base and the height of a triange and returns the area. The area of a triange is the base times the height divided by two.
+// -----COMPLETE------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
+// almost didnt add yarn to jest before running haha
+// write out basic form of the tests
+// // create sentence saying what I want "two numbers total"
+// // needs and argument and a return
+describe("addThemUp",() =>{
+  test("takes two numbers and returns the sum",()=>{
+    expect(addThemUp(2 + 3)).toEqual(5)
+  })
+})
+// needs a function to operate, DO NOT NEED console.log();
+// just yarn jest in terminal
+const addThemUp = () =>{
+  return "sum"
+}
```

Comment:
This function, as of right now, only returns back a string that says "sum". It also does not have placeholders such as (num1, num2).

It does not actually calculate anything. 

Did your test pass when you ran the test?